scripts/build.sh

```diff
@@ -78,6 +78,7 @@ build_operator_image() {
 
 build_bundle_image() {
   echo "Building and pushing $BUNDLE_IMAGE image"
+  operator-sdk generate bundle --operator-name awx-operator --version $TAG
```

Comment:
That is great so we can have the option to generate the bundle here as well!

Reply:
One less step when testing changes that affect the bundle :smile: